Add test output examples (#32)
Shimmur/checker_cab: README.md

values or if the view code does not capture newly-added fields to the `User`
schema.

For the sake of example, if the controller action under test returned a `User`
with the same keys and values differing only with the value for id, then the
test would fail and the output may look like the example below:

```
  1) test success: it returns a 200 and a newly updated User (UserControllerTest)
     test/user_controller_test.exs:294

>      There were issues the comparison:

Reviewer: Suggestion, blocking (but not for the reasons you would expect)

```suggestion
     There were issues with the comparison:
```
Through reviewing, I notice that there is a typo with the test output here. We're missing the word `with`.
[See the offending code here](https://github.com/Shimmur/checker_cab/blob/4a62ccbd07d103cc9967428f53b260e0bd2a2f26/lib/checker_cab.ex#L182)

This PR should not get merged in until #33 gets resolved (or it could be fixed in this PR).

     Values did not match for:
       field: :id
         expected: "1ee3e9c4-fa81-4612-a13a-c554e5a3d438"
         actual: "a2ab48fa-96fb-41e5-8f2d-2f94f47fef91"

>      code: assert_values_for(expected: expected_updates, actual: {returned_user :string_keys}, fields: fields_for(User))

Author: We replicated the output because we did not have the phoenix application where the demo test came from.

This faithfully copies the example code it refers to, but we do not know what lines ExUnit will truly return.

     stacktrace:
     ## stacktrace here
```

> As a different example, lets assume the same unit test. Through working a new
> feature, we add a `new_key` field to our `User` struct, but forget to add

Reviewer: Suggestion, phrasing:
```suggestion
As a different example, lets assume the same unit test. While adding a new
feature, we add a `new_key` field to our `User` struct, but forget to add
```

`new_key` to our `Factory` module. Checker Cab should detect the mismatch and
alert the developer with following output:

```
  1) test success: it returns a 200 and a newly updated User (UserControllerTest)
     test/user_controller_test.exs:294

>      There were issues the comparison:

Reviewer: Suggestion, blocking (see above for explanation / issue #33)
```suggestion
     There were issues with the comparison:
```
